databricks/koalas/plot/core.py

```diff
@@ -338,31 +344,41 @@ def _get_plot_backend(backend=None):
 
             KoalasPlotAccessor._backends["matplotlib"] = module
 
+        if backend == "plotly":
```

Comment:
Here I avoided to monkey patch plotly. I will need to refactor codes here in another PR .. here needs some refactoring.